app/components/AnalyzeComponent.js

```diff
@@ -32,60 +30,35 @@ interface Props {
   jupyterActions: Object;
 }
 
-interface State {
-  eegFilePaths: Array<?{
-    key: string,
-    text: string,
-    value: { name: string, dir: string }
-  }>;
-  selectedFilePaths: Array<?string>;
-}
-
 export default class Analyze extends Component<Props, State> {
   props: Props;
-  state: State;
   handleDropdownChange: (Object, Object) => void;
   handleLoadData: () => void;
+  handleAnalyze: () => void;
   handleChannelDropdownChange: (Object, Object) => void;
 
   constructor(props: Props) {
     super(props);
-    this.state = {
-      eegFilePaths: [{ key: "", text: "", value: { name: "", dir: "" } }],
-      selectedFilePaths: []
-    };
-    this.handleDropdownChange = this.handleDropdownChange.bind(this);
-    this.handleLoadData = this.handleLoadData.bind(this);
+    this.handleAnalyze = this.handleAnalyze.bind(this);
     this.handleChannelDropdownChange = this.handleChannelDropdownChange.bind(
       this
     );
   }
 
   componentWillMount() {
     if (isNil(this.props.kernel)) {
-      this.props.jupyterActions.launchKernel();
+      this.props.history.push("/clean");
```

Comment:
Let's replace this with something better. Analyze screen should display a recommendation to clean data rather than automatically redirect

Reply:
Replaced with a basic modal. Awaiting further design input